### PR TITLE
Adding Configurable Display type to Salesforce Auth Request

### PR DESCRIPTION
**Pull request by sprice88**
Allow for change between different display types. Added Enum to handle default, with values as specified [here in salesforce docs
](https://help.salesforce.com/articleView?id=remoteaccess_oauth_web_server_flow.htm&type=5)

### CODE REVIEW

File: src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationOptions.cs [hunk 1]
```diff
@@ -110,6 +125,10 @@ public string Caption
         ///     <see cref="System.Security.Claims.ClaimsIdentity" />.
         /// </summary>
         public string SignInAsAuthenticationType { get; set; }
+        /// <summary>
+        ///     Gets or sets the display—(Optional) 
+        /// </summary>
+        public Display DisplayMode { get; set; }
```
Comment (TerribleDev) [inline src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationOptions.cs]: This should default to page display mode

**Comment by TerribleDev** [comment]
Hi, 

sorry I havn't seen this PR. I was on vacation when you dropped this, and didn't notice until now.

File: src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationOptions.cs [hunk 2]
```diff
@@ -26,6 +26,21 @@ public class SalesforceAuthenticationEndpoints
...
+        /// <summary>
+        /// Options for Display Mode
+        ///     Changes the login and authorization pages’ display type. Salesforce supports these values.
+        ///     page—Full-page authorization screen(default)
+        ///     popup—Compact dialog optimized for modern web browser popup windows
+        ///     touch—Mobile-optimized dialog designed for modern smartphones, such as Android and iPhone
+        ///     mobile—Mobile-optimized dialog designed for less capable smartphones, such as BlackBerry OS 5
+        /// </summary>
+        public enum Display{
+            page,
```
Comment (TerribleDev) [inline src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationOptions.cs]: This should default to page

File: src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationOptions.cs (hunk 1 above)
Comment (TerribleDev) [inline src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationOptions.cs:131]: Shouldn't This should default to page?

**Review by TerribleDev — CHANGES_REQUESTED** [review]

**Comment by TerribleDev** [comment]
Hi, just one quick code comment if you don't mind...

File: src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationOptions.cs (hunk 1 above)
Comment (sprice88) [reply src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationOptions.cs:131]: Display is an enum with Page as the default value - I can make it explicit if you would prefer?

Comment (TerribleDev) [reply src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationOptions.cs:131]: No its fine thanks

Comment (sprice88) [reply src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationOptions.cs:131]: good to complete PR then?

**Pull request merged**